### ISSUE TITLE
Color constrast utility interface

### ISSUE THREAD

**Issue by kripod** [omgovich/colord#11]
Thank you for creating this awesome library!

I’ve been using color2k’s [`hasBadContrast` utility](https://color2k.com/#has-bad-contrast) for improving a11y lately. Unfortunately, it doesn’t support specifying a background color by nature, which is a dealbreaker for dark UIs.

Following an upcoming contrast measurement standard’s ideas, I think the contrast utility should have an interface where the desired contrast ratio/standard should be specified first, and then the text and the background color(s) should follow, as [explained in the SAPC-APCA contrast standard’s repository](https://github.com/Myndex/SAPC-APCA/blob/08a6387df53e93c7825009bd4583872b48687457/JS/ReadMe.md#important-do-not-mix-up-text-and-background-inputs).

**Comment by omgovich** [comment]
Hi! Work on the a11y/contrast tools is going to be started soon.

At the moment I think we re gonna rely on WGAC recommendations such as:
https://www.w3.org/TR/WCAG20-TECHS/G18.html#G18-tests

**Comment by omgovich** [comment]
Just released `v0.8` with the new plugin included.
